crates/gpui/src/platform/windows/platform.rs

```diff
@@ -81,6 +88,7 @@ pub(crate) struct WindowsPlatformInner {
     text_system: Arc<WindowsTextSystem>,
     callbacks: Mutex<Callbacks>,
     pub(crate) window_handle_values: RefCell<WindowHandleValues>,
+    pub raw_window_handles: RwLock<Vec<HWND>>,
```

Comment:
`window_handle_values` and `raw_window_handles` are substantially the same. We can merge them.